### PR TITLE
feat: set focus on nebula selection toolbar when tabbing

### PR DESCRIPTION
**Pull request by haxxmaxx**
Added `keyboard.focusSelection` of upcoming release of nebula. This enables us to focus one of the button in the selection toolbar. Since you navigate between the buttons in the selections toolbar with tab, this becomes a tab loop, which should be pretty intuitive for screen-reader users.

### CODE REVIEW

File: src/table/utils/handle-accessibility.js [hunk 1]
```diff
@@ -69,6 +69,13 @@ export const handleFocusoutEvent = (evt, shouldRefocus, blur) => {
   }
 };
 
-// get the object, find the patent of the confirm button in selection toolbar parent and focus that element
-export const focusConfirmButton = (element) =>
-  element.closest('.qv-object-wrapper')?.querySelector('.sel-toolbar-confirm')?.parentElement?.focus();
+export const focusSelectionToolbar = (element, keyboard, last) => {
+  const clientConfirmButton = element
```
Comment (haxxmaxx) [inline src/table/utils/handle-accessibility.js:73]: Don't have a better way of doing this. We could introduce some kind of inside client flag, but this works just fine

File: nebula.config.js [hunk 2]
```diff
@@ -10,4 +10,7 @@ module.exports = {
   build: {
     replacementStrings,
   },
+  serve: {
+    keyboardNavigation: false,
```
Comment (haxxmaxx) [inline nebula.config.js:14]: This can be set to true to test the "sheet mode"

File: src/table/utils/handle-accessibility.js (hunk 1 above)
Comment (niekvanstaveren) [reply src/table/utils/handle-accessibility.js:73]: Which confirm button is this? Would it not make sense if this is also exposed in keyboard?

Comment (haxxmaxx) [reply src/table/utils/handle-accessibility.js:73]: This is the confirm button in the client selection toolbar. The reason why we can focus the nebula one is that we can access the keyboard hook. keyboard don't have access to the client one

**Review by niekvanstaveren — APPROVED** [review]

**Pull request merged**